### ISSUE TITLE
Document macOS localization

### ISSUE THREAD

**Issue by stuartmorgan** [flutter/website#7767]

**Comment by atsansone** [comment]
@stuartmorgan : Are these changes similar to the [iOS process](https://docs.flutter.dev/accessibility-and-localization/internationalization#localizing-for-ios-updating-the-ios-app-bundle)? Could you point out some resources?

**Comment by stuartmorgan** [comment]
Yes, macOS also defines supported localizations via the Info.plist

**Comment by sfshaza2** [comment]
Wow, this is ancient. @stuartmorgan, can this be as simple as updating the header to "iOS and macOS"?
